src/transaction/combine.ts

```diff
@@ -1,42 +1,80 @@
 import * as _ from 'lodash'
 import binary from 'ripple-binary-codec'
-import * as utils from './utils'
 import BigNumber from 'bignumber.js'
+import {ValidationError} from '../common/errors'
 import {decodeAccountID} from 'ripple-address-codec'
 import {validate} from '../common'
 import {computeBinaryTransactionHash} from '../common/hashes'
+import {JsonObject} from 'ripple-binary-codec/dist/types/serialized-type'
+
+//The transactions should all be equal EXCEPT for the Signers field
+function validateAllSignedTransactionsAreEqual( transactions : Array<JsonObject> ) {
+  const tx = transactions[0];
+  const _txSignersTemp = tx.Signers;
+  delete tx.Signers
+  
+  transactions.forEach( _tx => {
+      const _txSignersTemp = _tx.Signers
+      delete _tx.Signers;
+      
+      if ( JSON.stringify(tx) !== JSON.stringify(_tx)) {
+          throw new ValidationError('txJSON is not the same for all signedTransactions')
+      }
+
+      _tx.Signers = _txSignersTemp
+  })
+
+  tx.Signers = _txSignersTemp
+  return transactions
+}
 
 function addressToBigNumber(address) {
   const hex = Buffer.from(decodeAccountID(address)).toString('hex')
   return new BigNumber(hex, 16)
 }
 
+/**
+ * According to the documentation on multi-signatures:
+ * If presented in binary form, the Signers array must be sorted based on 
+ * the numeric value of the signer addresses, with the lowest value first. 
+ * (If submitted as JSON, the submit_multisigned method handles this automatically.)
+ */
 function compareSigners(a, b) {
   return addressToBigNumber(a.Signer.Account).comparedTo(
     addressToBigNumber(b.Signer.Account)
   )
 }
 
-function combine(signedTransactions: Array<string>): object {
-  validate.combine({signedTransactions})
-
-  // TODO: signedTransactions is an array of strings in the documentation, but
-  // tests and this code handle it as an array of objects. Fix!
-  const txs: any[] = _.map(signedTransactions, binary.decode)
-  const tx = _.omit(txs[0], 'Signers')
-  if (!_.every(txs, (_tx) => _.isEqual(tx, _.omit(_tx, 'Signers')))) {
-    throw new utils.common.errors.ValidationError(
-      'txJSON is not the same for all signedTransactions'
-    )
-  }
+function getATransactionWithAllSigners(transactions : Array<JsonObject>) : JsonObject {
   const unsortedSigners = _.reduce(
-    txs,
+    transactions,
     (accumulator, _tx) => accumulator.concat(_tx.Signers || []),
     []
   )
-  const signers = unsortedSigners.sort(compareSigners)
-  const signedTx = _.assign({}, tx, {Signers: signers})
-  const signedTransaction = binary.encode(signedTx)
+  //The documentation requires signers to be sorted
+  const sortedSigners = unsortedSigners.sort(compareSigners)
+
+  //This is used to make a new copy of the transaction with ALL signers
+  const signedTx = _.assign({}, transactions[0], {Signers: sortedSigners})
```

Comment:
Can we use `Object.assign()` here

Reply:
there's no benefit to assigning `unsortedSigners` to a variable, since it will only be used once. I think that makes your code unnecessarily verbose. instead:

```javascript
function getATransactionWithAllSigners(transactions : Array<JsonObject>) : JsonObject {
  // Signers must be sorted
  const sortedSigners = transactions
    .map(tx => tx.Signers)
    .filter(signer => signer)
    .sort(compareSigners)

  return Object.assign({}, {...transactions[0], Signers: sortedSigners})
}
```

Reply:
That looks cleaner to me :) 
(Unfortunately I still had to use lodash to get flatMap as it isn't built in, but it's much more readable than assign!)

Reply:
It's totally fine to use lodash's `flatMap`, it's used elsewhere too I think